framework/src/main/java/fi/otavanopisto/pyramus/applications/ApplicationUtils.java

```diff
@@ -830,13 +840,43 @@ public static void sendNotifications(Application application, HttpServletRequest
           staffMember);
     }
   }
+  
+  public static String extractBirthdayString(Application application) {
+    if (application == null) {
+      return null;
+    }
+    JSONObject formData = JSONObject.fromObject(application.getFormData());
+    String ssn = getFormValue(formData, "field-ssn");
+    if (!StringUtils.isEmpty(ssn)) {
+      int d = Integer.parseInt(ssn.substring(0, 2));
+      int m = Integer.parseInt(ssn.substring(2, 4));
+      int y = Integer.parseInt(ssn.substring(4, 6));
```

Comment:
parseInt might throw an exception here, are we relying on frontend validation or should it be catched?

Reply:
Same for charAt below, it might throw an exception too